DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Raw/BoundarySegmentConversionToSpeckle.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Objects;
+using Speckle.Converters.Common.Objects;
+
+namespace Speckle.Converters.RevitShared.ToSpeckle;
+
+public class BoundarySegmentConversionToSpeckle : IRawConversion<IList<DB.BoundarySegment>, SOG.Polycurve>
+{
+  private readonly IRawConversion<DB.Curve, ICurve> _curveConverter;
+
+  public BoundarySegmentConversionToSpeckle(IRawConversion<DB.Curve, ICurve> curveConverter)
+  {
+    _curveConverter = curveConverter;
+  }
+
+  public SOG.Polycurve RawConvert(IList<DB.BoundarySegment> target)
+  {
+    if (!target.Any())
+    {
+      throw new ArgumentException("Input Boundary segment list must at least have 1 segment");
```

Comment:
Related to a comment on Ian's PR. Is this supposed to be a SpeckleConversionException?

Reply:
I don't think it should... we didn't actually "failed to convert" but rather "the input provided was not valid" which is not really the same.

I'm not very particular doing one or the other, we need to decide on a consistent strategy for this.